examples/keystone-in-nextjs/src/keystone/schema.ts

```diff
@@ -0,0 +1,38 @@
+import { list } from '@keystone-6/core';
+import { text, password, timestamp } from '@keystone-6/core/fields';
+import type { Lists } from '.keystone/types';
+
+const permissions = {
+  authenticatedUser: ({ session }: any) => !!session?.data,
+  public: () => true,
+  readOnly: {
+    operation: {
+      query: () => true,
```

Comment:
```suggestion
      query: allowAll,
```

Reply:
Yup. I have some changes in local. Will push this along with that.